stdlib/public/SDK/Foundation/FileManager.swift

```diff
@@ -45,4 +52,16 @@ extension FileManager {
         }
         throw error!
     }
+    
+    @available(OSX 10.6, iOS 4.0, *)
+    public func enumerator(at url: URL, includingPropertiesForKeys keys: [URLResourceKey]?, options mask: FileManager.DirectoryEnumerationOptions = [], errorHandler handler: ((URL, Error) -> Bool)? = nil) -> FileManager.DirectoryEnumerator? {
```

Comment:
Ah, sorry, this should probably be marked `final` and `@nonobjc` (per discussion with @parkera).